### ISSUE TITLE
Multiple VS Code projects cause wrong identification

### ISSUE THREAD

**Issue by muuvmuuv** [onlyutkarsh/git-config-user-profiles#95]
**Describe the bug**

It is just visual but when opening two projects one will show the wrong ID in the bottom bar.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open two VS Code projects
2. Bottom bar shows wrong ID

**Expected behaviour**

**Screenshots**
If applicable, add screenshots to help explain your problem. Pasting the text from output window helps to troubleshoot issues.

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10, Ubuntu]

**Additional context**

I think if you would create a hash from all values that are saved to git-user you can better compare local and saved configs. Instead of using a generated uuid.

**Comment by onlyutkarsh** [comment]
You're absolutely right, and thank you for raising this issue! The current behavior of storing the selected profile in workspace settings (.vscode/settings.json) doesn't make sense and causes problems. The profiles themselves (with username/email) are already stored in user/global settings - they're personal to each developer. The selected profile ID is just a reference to your personal profiles, so there's no reason to share it with the team. 

I'll see if I can improve it.

**Comment by onlyutkarsh** [comment]
I have fixed this in v2.3.0. The existing setting in `gitConfigUser.selectedProfileId` will be automatically migrated.
